stock_report_quantity_by_location/wizards/stock_report_quantity_by_location.py

```diff
@@ -69,6 +73,7 @@ def _compute_stock_report_by_location(self):
                             "quantity_on_hand": qty_on_hand,
                             "quantity_reserved": qty_reserved,
                             "quantity_unreserved": qty_unreserved,
+                            "quantity_available": product.virtual_available,
```

Comment:
@enriquemartin Why not adding the field to the read_group ? With this, you will impact performances as qty_available function will be called twice

Reply:
> 

Have I answered your doubts @rousseldenis?